docs/scripts/fledge_plugin_list

```diff
@@ -96,6 +96,103 @@ function table {
     done
 }
 
+function keywordheader {
+    keyfile="$1"
+    title=$2
+    echo ".. list-table:: ${title}" >> "$keyfile"
+    echo "    :widths: 20 50" >> "$keyfile"
+    echo "    :header-rows: 1" >> "$keyfile"
+    echo "" >> "$keyfile"
+    echo "    * - Name" >> "$keyfile"
+    echo "      - Description" >> "$keyfile"
+}
+
+function keywords {
+    list="$1"
+    title=$2
+    tableType=$3
+    if [[ -d /tmp/keywords ]]; then
+	rm -rf /tmp/keywords
+    fi
+    mkdir /tmp/keywords
+    for repo in ${list}
+    do
+        type=$(echo "${repo}" | sed -e 's/fledge-//' -e 's/-.*//')
+        name=$(echo "${repo}" | sed -e 's/fledge-//' -e "s/${type}-//")
+        if [[ ${type} = "$tableType" ]]; then
+            if grep -q "$repo" <<< "$INBUILT_PLUGINS"; then
+                if [[ $repo == "fledge-north-OMF" ]]; then
+			keyword=OMF
+			keyfile="/tmp/keywords/${keyword}"
+			if [[ ! -f "${keyfile}" ]]; then
+			    keywordheader "$keyfile" "$title"
+			fi
+			    if [[ -f ${repo}/docs/index.rst ]]; then
+                                echo "    * - \`$name <"plugins/fledge-north-OMF/index.html">\`__" >> "$keyfile"
+			     else
+                                echo "    * - ${name}" >> "${keyfile}"
+			    fi
+                            echo "      - ${description}" >> "${keyfile}"
+                elif [[ $repo == "fledge-rule-DataAvailability" ]]; then
+			echo TODO deal with inbuilt data availability plugin
+                elif [[ $repo == "fledge-rule-Threshold" ]]; then
+			echo TODO deal with inbuilt threshold plugin
+                fi
+                else
+                    rm -rf ${repo}
+                    git clone https://${USERNAME}:${GITHUB_ACCESS_TOKEN}@github.com/fledge-iot/${repo}.git --branch ${DOCBRANCH} >/dev/null 2>&1
```

Comment:
Pulling the repository twice will add unnecessary time. We should utilize the existing `function table` to handle this more efficiently.